docs/theming/customizing.md

```diff
@@ -72,15 +90,23 @@ and to both the `light` and `dark` backgrounds.
 
 ## Custom themes
 
-To create a custom theme, designers and developers need only set the values for
-the relevant color properties in the design system. For example, a _Bordeaux_
-theme might look like this:
+To create a custom theme, designers and developers need only set the values for 
+the relevant color properties in the design system in their page's <abbr 
+  title="Cascading style sheets">CSS</abbr>. For example, let's create a custom 
+_Bordeaux_ theme. We'll write a CSS rule which selects for the `theme-bordeaux` 
+class, and in that rule we'll set the values of <abbr title="red hat design 
+  system">RHDS</abbr> theming tokens to our custom theme colors.
+In the following example, you can see how we first define some custom theme 
+colours as CSS custom properties beginning with `--bordeaux-`, then we use those 
+values in our custom theme by applying them to RHDS theming tokens, i.e. tokens 
+that look like `--rh-color-*-on-(light|dark)`.
```

Comment:
Should we add an alert here noting the `-on-light|dark` tokens are deprecated and to use the `light-dark()` function as seen below?

Reply:
Also see lower on the page the `Careful` alert noting the `-on-light|dark` being used to calculate the ones we are now actively overwriting with `light-dark()`

Reply:
Good catch. fixed.